src/examples/eigen_matrix.cpp

```diff
@@ -1,50 +0,0 @@
-/*
- *  Copyright (c), 2017, Adrien Devresse
- *  Copyright (c), 2022, Blue Brain Project
- *
- *  Distributed under the Boost Software License, Version 1.0.
- *    (See accompanying file LICENSE_1_0.txt or copy at
- *          http://www.boost.org/LICENSE_1_0.txt)
- *
- */
-#include <iostream>
-
-#ifdef H5_USE_EIGEN
-
-#include <Eigen/Dense>
-#include <highfive/H5File.hpp>
-
-using namespace HighFive;
-
-const std::string FILE_NAME("eigen_matrix_example.h5");
-const std::string DATASET_NAME("dset");
-const int nrows = 10;
-const int ncols = 3;
-
-// Create a 2D dataset 10x3 of double with eigen matrix
-// and write it to a file
-int main(void) {
-    try {
-        Eigen::MatrixXd matrix(nrows, ncols);
```

Comment:
In stead of removing, you could use a row-major example for the moment. We can then revert one someone had the time to fix HighFive

Reply:
No, `git revert` works quite well. I don't want there to be anything suggesting that reading or writing `Eigen::Matrix` from core HighFive is sensible.